code/__DEFINES/hud_locations.dm

```diff
@@ -77,11 +77,11 @@
 #define ui_borg_lanugage_menu "EAST-2:26,SOUTH+1:7"
 
 //Upper-middle right (alerts)
-#define ui_alert1 "EAST-1:28,CENTER+5:27"
-#define ui_alert2 "EAST-1:28,CENTER+4:25"
-#define ui_alert3 "EAST-1:28,CENTER+3:23"
-#define ui_alert4 "EAST-1:28,CENTER+2:21"
-#define ui_alert5 "EAST-1:28,CENTER+1:19"
+#define ui_nutrition "EAST-1:28,CENTER+5:27"
```

Comment:
IMO, we keep adding new alerts, we need to keep this alert slot. Could we move the hunger indicator somewhere else? Maybe below the stamina bar?

Reply:
Well, nutrition alert always were occupying that slot unless you are a human. But I'll see where else it could be placed...

Reply:
The stamina bar also won't be visible all the time, so it'll mostly have a big gap between it and the health doll usually
Imo, fine to stay where it currently is

Reply:
We could place it to the left of the alerts, not all mobs have nutrition, but they do all get alerts

Reply:
What about left of healthdoll?

Or we could move it 1 spot up, aligned to action button's 1 row?